lambda-extensions/sumoclient/sumoclient.go

```diff
@@ -179,6 +179,7 @@ func (s *sumoLogicClient) getLogStream() string {
 
 func (s *sumoLogicClient) enhanceLogs(msg responseBody) {
 	s.logger.Debugln("Enhancing logs")
+	var jsonMessage interface{}
 	for _, item := range msg {
 		// item["FunctionName"] = s.config.FunctionName
```

Comment:
Why keep this? We could overwrite the whole thing and make logs coming from here to look like normal logs

Reply:
okay, let me try to simplify it.

Reply:
updated 4d7cd56